gh-deployments-environments/helper.sh

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env bash
+
+set -e
+
+CMD=${1:-};
+
+
+print_syntax() {
+  echo "helper.sh <command> [args...]"
+  echo ""
+  echo "Required environment variables:"
+  echo "  GITHUB_TOKEN  GitHub token used when calling API"
+  echo ""
+  echo "Available commands:"
+  echo "  help"
+  echo "    Prints this screen"
+  echo ""
+  echo "  create-deployment <owner> <repo> <env> <ref> <output-file-for-id>"
+  echo "    Creates new deployment"
+  echo ""
+  echo "  create-deployment-status <owner> <repo> <deployment-id> <status>"
+  echo "    Creates deployment status"
+  echo ""
+  echo "  delete-environment <owner> <repo> <env>"
+  echo "    Deletes environment and all deployments and statuses related to it"
+  echo ""
+}
+
+
+check_env_vars() {
+	if [[ -z "${GITHUB_TOKEN}" ]]; then
```

Comment:
Fix indentation please

Reply:
Fixed.  I'll resolve this conversation, it's approved and I'd like to use that tiny github action already in one tiny workflow we have that tidies up featurenets.